person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+
+  attr_accessor :name, :age
```

Comment:
- [ OPTIONAL BUT HIGHLY RECOMMENDED ] Hello! It's often considered a best practice to place Getters and Setters at the top of a class definition. 😊 Your current placement has them in the body of the class. What do you think about relocating them to the top? 🤔 This can help improve the readability and organization of your code, making it easier to understand and maintain. 🚀